.gitignore

```diff
@@ -37,3 +37,6 @@ Brewfile.lock.json
 .env
 .env.local
 .env.**.local
+
+# ignore simplecov output
+/coverage
```

Comment:
also add to `.dockerignore` 
[example](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/.dockerignore#L21)

Reply:
forget that, already in `.dockerignore`